useful/template.live_variables.js

```diff
@@ -1,18 +1,25 @@
 module.exports = {
-  cfunctions_path: "./server/common_functions.js",
-  functions_path: "./server/server_functions.js",
-  worker_path: "./server/server_worker.js",
-  data_path: "./server/data.js",
-  base_url: "https://yourappspoturldontincludedashes.appspot.com",
-  keyword: "123",
-  access_master: "123",
-  bot_key: "123",
-  discord_token: "NDXXXXXXXXXXX...",
-  apple_token: "acXXXXXXXX...",
-  steam_key: "8aXXXXXXXXX...",
-  steam_web_key: "B4XXXXXXX...",
-  steam_partner_key: "F9XXXXXXX...",
-  close_timeout: 24000,
-  ip_limit: 3,
-  character_limit: 3,
+	cfunctions_path: "./server/common_functions.js",
+	functions_path: "./server/server_functions.js",
+	worker_path: "./server/server_worker.js",
+	data_path: "./server/data.js",
+	base_url: "https://yourappspoturldontincludedashes.appspot.com",
+	keyword: "123",
+	access_master: "123",
+	bot_key: "123",
+	apple_token: "acXXXXXXXX...",
+	steam_key: "8aXXXXXXXXX...",
+	steam_web_key: "B4XXXXXXX...",
+	steam_partner_key: "F9XXXXXXX...",
+	close_timeout: 24000,
+	ip_limit: 3,
+	character_limit: 3,
+	DISCORD: {
+		ENABLED: false,
+		TOKEN: "NDXXXXXXXXXXX",
```

Comment:
Duplicated token? Why is this here as well? (Maybe you don't even know)

Reply:
The live variables is used as part of the deploy process on the official servers as far as I understand, there is a script that copies this and some other folders into the deployed directory.